### ISSUE TITLE
Who is using Chatto?

### ISSUE THREAD

**Issue by diegosanchezr** [badoo/Chatto#211]
We'd love to know about apps in the store using the framework.

Feel free to say it out loud here; or contact me privately by [email](mailto:diego.sanchez@corp.badoo.com) or [Twitter](https://twitter.com/spaxeon)

Thank you!

**Comment by zwang** [comment]
@diegosanchezr  Our app has not been released yet. Still working on it. Once it is released, I will update here. Thanks for the library and actively improving it! :)

**Comment by weyert** [comment]
Yes, same here still working on the application. It's a great library and I enjoy using it. Wish I could just work on my app every workday so it would be finished by now :)

**Comment by seidju** [comment]
We're also making some kind of messenger based on XMPPFramework and this great framework! Hope will release it soon!

**Comment by zwang** [comment]
@diegosanchezr  just want to do a quick update: Our app has been released. The chat views are all based on Chatto. Thank you for the great library. Really appreciate it. :)
